base/sysinfo.jl

```diff
@@ -321,4 +323,91 @@ windows_version
 
 const WINDOWS_VISTA_VER = v"6.0"
 
+"""
+    Sys.isexecutable(path::AbstractString)
+
+Returns `true` if the given `path` has executable permissions.
+"""
+function isexecutable(path::AbstractString)
+    @static if iswindows()
+        return isfile(path)
```

Comment:
Can't you determine whether it has executable permissions using `_access` on Windows?

Reply:
Nope.  Windows doesn't have an `X_OK` bit.  :(

https://msdn.microsoft.com/en-us/library/1w06ktdy.aspx

Reply:
Ah okay, sorry for the noise then.